tests/unit/store-test.js

```diff
@@ -120,6 +120,102 @@ test("models with str based ids must be lookedup by str value", function(assert)
   assert.ok(toranbByStr.get("content") instanceof Person);
 });
 
+test("models with int !0 id must be lookedup by string value", function(assert) {
```

Comment:
we do have a test in the suite that still passes that has this as the name

`models with int based ids must be lookedup by int value`

Does it make sens to rename this as it would appear you can lookup "person" wi3th both "1234" and 1234 - am I incorrect? Below is that full test I'm making reference to 

```js
test("models with int based ids must be lookedup by int value", function(assert) {
  store.push("person", {
    id: 123,
    firstName: "Toran",
    lastName: "Billups"
  });

  var toranbByNum = store.find("person", 123);
  assert.strictEqual(toranbByNum.get("id"), 123);
  assert.ok(toranbByNum.get("content") instanceof Person);
});
```